UnityWebRequest.isHttpError' is obsolete:

I noticed these messages when upgrading to 2020

Assets/UnityMarkdownViewer/Editor/Scripts/MarkdownHandleImages.cs(159,17): warning CS0618: 'UnityWebRequest.isHttpError' is obsolete: 'UnityWebRequest.isHttpError is deprecated. Use (UnityWebRequest.result == UnityWebRequest.Result.ProtocolError) instead.'

I also get this, but I don't know if it's related to UMV:

Failed to destroy editor windows: #1
FallbackEditorWindow
UnityEditor.WindowLayout:LoadDefaultWindowPreferences () (at /home/bokken/buildslave/unity/build/Editor/Mono/GUI/WindowLayout.cs:72)

--- a/Editor/Scripts/MarkdownHandleImages.cs
+++ b/Editor/Scripts/MarkdownHandleImages.cs
@@ -156,12 +156,20 @@ namespace MG.MDV
                 return false;
             }
 
+#if UNITY_2020_2_OR_NEWER
+            if( req.Request.result == UnityWebRequest.Result.ProtocolError )
+#else
             if( req.Request.isHttpError )
+#endif
             {
                 Debug.LogError( string.Format( "HTTP Error: {0} - {1} {2}", req.URL, req.Request.responseCode, req.Request.error ) );
                 mTextureCache[ req.URL ] = null;
             }
+#if UNITY_2020_2_OR_NEWER
+            else if( req.Request.result == UnityWebRequest.Result.ConnectionError )
+#else
             else if( req.Request.isNetworkError )
+#endif
             {
                 Debug.LogError( string.Format( "Network Error: {0} - {1}", req.URL, req.Request.error ) );
                 mTextureCache[ req.URL ] = null;

--- a/Editor/Scripts/MarkdownImporter.cs
+++ b/Editor/Scripts/MarkdownImporter.cs
@@ -1,7 +1,11 @@
 ﻿#if UNITY_2018_1_OR_NEWER
 
 using UnityEngine;
+#if UNITY_2020_2_OR_NEWER
+using UnityEditor.AssetImporters;
+#else
 using UnityEditor.Experimental.AssetImporters;
+#endif
 using System.IO;
 
 namespace MG.MDV

